api/config.proto

```diff
@@ -30,6 +30,8 @@ message Repo {
     Auth auth = 3;
     // The path where the repo stores charts. Useful for LOCAL kind only
     string path = 4;
+    // The uri where the OCI repository keeps the index of charts
+    string ociIndex = 5;
```

Comment:
I would rename this to `chartsIndex` because OCI is just one implementation where we are allowing users to store this in-house index. We could potentially add/retrieve this index to helm classic registries if we want.

Reply:
Done! The same was suggested by @migmartri